lib/modules/datasource/docker/schema.ts

```diff
@@ -127,3 +130,35 @@ export const Manifest = z.union([
 ]);
 
 export type Manifest = z.infer<typeof Manifest>;
+
+export const DockerHubTag = z
+  .object({
+    name: z.string(),
+    tag_last_pushed: z.string().nullable().catch(null),
```

Comment:
```suggestion
    tag_last_pushed: z.string().datetime().nullable().catch(null),
```
should be a valid iso timestamp